src/components/pieces/PiecesComponent.js

```diff
@@ -32,11 +36,17 @@ export default class PiecesComponent extends React.Component {
                     <Toggle checked={this.props.editorActive}
                             onChange={() => this.props.piecesToggleEdit(false)}/>
                 </div>
-                {   Object.keys(this.props.components).map((object, index) => <div className="r_list-header r_list-subheader" key={index}>
-                    <label>{this.props.components[object].__name || object}</label>
-                    <Toggle checked={this.props[`editorEnabled:${object}`]}
-                            onChange={() => this.props.piecesToggleEdit(object)}/>
-                </div>) }
+                {   Object.keys(this.props.components).map((object, index) => {
+                    if(! representPieceTypes[object]){
+                        return '';
+                    }
+
+                    return (<div className="r_list-header r_list-subheader" key={index}>
```

Comment:
in ReactJS docs conditional rendering has following way:
`return representPieceTypes[object] && (<div className="r_list-header r_list-subheader" key={index}...........`
returning null or false or undefined equals to rendering nothing